bitbots_quintic_walk/launch/quintic_walk.launch

```diff
@@ -15,6 +16,7 @@
             <rosparam command="load" file="$(find bitbots_bringup)/config/game_settings.yaml" />
         </group>
   </group>   
-    
-  <node name="walking" pkg="bitbots_quintic_walk" type="WalkNode" output="screen"/>
+  <group unless="$(arg python)">
```

Comment:
If python is true, no walk node will be started, I'm not sure whether that is intended.

Reply:
It is intended, but a bit hacky. In order to run the engine directly from python, a few things still need to be put on the parameter server, but you don't want to start the node since you do this directly from python.

Reply:
Maybe it should be renamed to something like `no_node` then?